src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -311,6 +311,52 @@ the library you can do so as follows:
 });
 ```
 
+#### Enriching ASP.NET Core Request Metrics
+
+ASP.NET Core supports enriching request metrics using `IHttpMetricsTagsFeature`.
+This feature allows you to add custom tags to metrics like `http.server.request.duration`,
+which records the duration of HTTP requests on the server.
+
+Here's an example of enriching the `http.server.request.duration` metric:
```

Comment:
where do user specify that they intend to enrich `http.server.request.duration` metric, and nothing else?

Reply:
Fixed in 99a4211340adb3c944e2380f9f010a72d6933dac

Reply:
I am still confused. Where does user specify that they intend to enrich "http.server.request.duration" metric, and nothing else?
The example just shows how to get `IHttpMetricsTagsFeature` and add tags to it. But don't see any place where user selects which metric is this applied to.